pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -154,7 +154,7 @@ func TestPostFilter(t *testing.T) {
 				"node1": framework.NewStatus(framework.UnschedulableAndUnresolvable),
 			},
 			wantResult: nil,
-			wantStatus: framework.NewStatus(framework.Unschedulable),
+			wantStatus: framework.NewStatus(framework.Unschedulable, "0/1 nodes are available: 1 Preemption is not helpful for scheduling."),
```

Comment:
Can you add a test like: 2 nodes failed on nodesWherePreemptionMightHelp() (due to UnschedulableAndUnresolvable status), and 2 nodes failed in dryRunPreemption()?

Reply:
Done